README.md

```diff
@@ -1,24 +1,87 @@
-# tr1d1um
+# Tr1d1um
 
 [![Build Status](https://travis-ci.org/Comcast/tr1d1um.svg?branch=master)](https://travis-ci.org/Comcast/tr1d1um) 
```

Comment:
```suggestion
[![Build Status](https://travis-ci.org/xmidt-org/tr1d1um.svg?branch=master)](https://travis-ci.org/xmidt-org/tr1d1um) 
```

Reply:
@kcajmagic, I don't think we can make this change until we port over travis itself as `https://travis-ci.org/xmidt-org/tr1d1um` doesn't work. I'll ask @schmidtw about this. I think he has the travis admin permissions.